pom.xml

```diff
@@ -2,12 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <packaging>pom</packaging>
-
-    <modules>
-        <module>template</module>
-        <module>database</module>
-    </modules>
+    <packaging>jar</packaging>
```

Comment:
mvn clean install
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for edition.academy.seventh:robot:jar:0.1
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.testng:testng:jar -> version 6.14.3 vs ${version.testng} @ line 100, column 21
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 


2 wersje TestNG!!!

Reply:
Linia 11 <relativePath/> co to jest??? mvn clean verify site ma problemy z budowaniem i wyrzuca następujące ostrzeżenie:
[WARNING] Unable to create Maven project for org.postgresql:postgresql:jar:42.2.5 from repository.
org.apache.maven.project.ProjectBuildingException: Some problems were encountered while processing the POMs:
[ERROR] Unknown packaging: bundle @ line 11, column 14
Caused by: org.apache.maven.model.building.ModelBuildingException: 1 problem was encountered while building the effective model for org.postgresql:postgresql:42.2.5
[ERROR] Unknown packaging: bundle @ line 11, column 14